src/main/java/org/folio/innreach/domain/entity/ItemContributionOptionsConfiguration.java

```diff
@@ -0,0 +1,72 @@
+package org.folio.innreach.domain.entity;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.CollectionTable;
+import javax.persistence.Column;
+import javax.persistence.ElementCollection;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.MapsId;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+@Entity
+@Table(name = "item_contribution_options_configuration")
+@Getter
+@Setter
+@ToString(exclude = {"centralServer", "statuses", "loanTypes", "locations", "materialTypes"})
+@EqualsAndHashCode(of = {"id"})
+public class ItemContributionOptionsConfiguration {
```

Comment:
All entities have to be auditable, meaning any change should be tracked with timestamp of the event and user.
The class has to extend `Auditable`